udata_front/frontend/helpers.py

```diff
@@ -211,6 +218,18 @@ def owner_name_acronym(obj):
     return ''
 
 
+@front.app_template_global()
+def organization_type(org: Organization):
+    if org.public_service:
+        return PUBLIC_SERVICE
+    elif org.local_authority:
+        return LOCAL_AUTHORITY
+    elif org.association:
+        return ASSOCIATION
+    else:
+        return COMPANY
```

Comment:
I think the issue with this approach is that a new organization without badge will be shown as `company` (not only the badge but the wording on its page) by default, which is wrong.

Reply:
I didn't realise we wanted a case without any type. If we do, we might want to make a new datagouv/components before this PR to change the condition [there too](https://github.com/datagouv/udata-front/blob/feat/show-organization-type/udata_front/theme/gouvfr/datagouv-components/src/composables/organizations/useOrganizationType.ts#L44).

Reply:
TS changes done in https://github.com/datagouv/udata-front/pull/475